sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -82,8 +82,37 @@ enum class graph_state {
   executable, ///< In executable state, the graph is ready to execute.
 };
 
+enum class node_type {
+  empty = 0,
+  subgraph,
+  kernel,
+  memcpy,
+  memset,
+  memfill,
+  prefetch,
+  memadvise,
+  ext_oneapi_barrier,
+  host_task
```

Comment:
```suggestion
  empty = 0,
  subgraph = 1,
  kernel = 2,
  memcpy = 3,
  memset = 4,
  memfill = 5,
  prefetch = 6,
  memadvise = 7,
  ext_oneapi_barrier = 8,
  host_task = 9
```
We might want to explicitly set integer values here.